book.rb

```diff
@@ -0,0 +1,16 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def add_rental(person, date)
+    rental = Rental.new(self, person, date)
+    @rentals << rental
+    person.add_rental(rental)
+  end
```

Comment:
![image](https://github.com/rmiyoyo/oop-school-library/assets/93039359/56126886-c76c-4428-979c-c151bb516727)

- [ ] Great work so far but when I tried to add the rental in the book then I see the above error :eyes: Kindly fix that error to make your app error free :+1: